### DISCUSSION TITLE
View annotated tag information

### DISCUSSION THREAD

**Discussion by BeWae** [gitextensions/gitextensions#10141] [Q&A]
Hi im Working with the newest Version of GitEx and i'm looking for the option to enable or view annotated Tag information.
The only thing  i can see is the message in the Commitwindow below.

Is there a configuration to see those information?

Cheers.

**Comment by pmiossec** [comment]
Normally the content is displayed at the end of the commit message of the
commit pointed by the tag.

You don't see it?

**Reply by BeWae** [reply]
Hi, maybe i get u wrong but what i mean is Information about the tag creator not the commit creator.

This is a annoted tag that stores tagger information
![at](https://user-images.githubusercontent.com/24584782/183596604-6449ec6b-f01b-49f9-ae11-8305e2f3f82b.png)

Here is a lightweight tag
![lw](https://user-images.githubusercontent.com/24584782/183596773-2d18c33d-ea5e-450e-996b-df64d3267008.png)

Cheers

**Comment by gerhardol** [comment]
There is no possibility to see the annotated tag info in GE now.
To handle this as a feature request that someone (maybe yourself?) can look into, can you provide some more information/suggestions:

* How should this info be displayed?
  - In commit info as additional info for each tag? Always visible or configurable? Tooltip? Need to copy?
  - Info in side panel?
  - How can this information be added in a way to not slow down the UI?

* What Git command can give this information in a format that can be safely parsed?

All information will not be visible in the UI...

**Comment by BeWae** [comment]
Hi, thanks for your reply.
I started my education a few weeks ago, so I can't answer your questions in a helpful way.
So I don't see myself in implementing that feature soon.
I forked and imported the projekt to VS and I was kinda overwhelmed 😄 .

The idea that I have is, to see the autor/date above the tag message in the commit tab.
If you have multiple tags, that could be a bit stuffed so a tab for tags could be also a thing.
Theoretically, you can include the option that you can choose yourself which view you want to have.
But is it worth doing that? - Maybe not.

The command that I use to see information about the tag is `git show <tag>` .
On the Website about [Tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) I could not find a better solution.

**Reply by gerhardol** [reply]
> Hi, thanks for your reply. I started my education a few weeks ago, so I can't answer your questions in a helpful way. So I don't see myself in implementing that feature soon. I forked and imported the projekt to VS and I was kinda overwhelmed 😄 .

You have to start somehow.

> The idea that I have is, to see the autor/date above the tag message in the commit tab. If you have multiple tags, that could be a bit stuffed so a tab for tags could be also a thing. Theoretically, you can include the option that you can choose yourself which view you want to have. But is it worth doing that? - Maybe not.

The more details you provide about how to present this and what the benefit is will increase the probability that someone is interested in this and supports and implements.

> The command that I use to see information about the tag is `git show <tag>` . On the Website about [Tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) I could not find a better solution.

That format does not seem to be safely parsable.
Many Git commands have "porcelain" outputs that are not changed from a release to another that can be used in a stable way.

In that case the output could of course be something like showing the command when clicking or something is a way.
